backend/pkg/config/proto_topic_mapping.go

```diff
@@ -14,8 +14,12 @@ package config
 // if the topics have been serialized without the schema registry being involved in the
 // serialization.
 type ProtoTopicMapping struct {
+	// TopicName is the name of the topic to apply these proto files. This supports regex.
 	TopicName string `yaml:"topicName"`
 
+	// IsRegex indicates if the topicName must be handled as a regex and match multiple topic names
+	IsRegex bool `yaml:"isRegex"`
```

Comment:
After all suggestions... 
**Why** can we not approach any topic name as a regex? 
**Why** do we need to increase the complexity of the mapping?

Let's try to compile **every `topicName` as regex**. 
For the Kafka name policy, it is completely fine!

If the topic name can not be compiled, let's put a warning to the logs and recognize such a `topicName` as a strict match to not break any exotic deployments.